src/pages/on-chain-governance/_components/ConfirmVoteDialog.tsx

```diff
@@ -477,3 +601,42 @@ function ReadyVoteID({
     </>
   );
 }
+
+function getVotesStyle(vote: VoteDecision | undefined) {
+  if (vote === VoteDecision.NO) {
+    return "text-red-600";
+  }
+  if (vote === VoteDecision.YES) {
+    return "text-green-600";
+  }
+  if (vote === VoteDecision.NEUTRAL) {
+    return "text-gray-600";
+  }
+}
+
+function setMasternodeOptions(
```

Comment:
avoid using `set...` as function name, because it is used for set{State} naming convention in react. Just use something explicit for onclick callback handler, e.g. `onContinueButtonClick`, `onVoteButtonClick`, etc